<launch>

    <!-- Load configuration parameters -->
    <rosparam 
        command="load" 
        file="$(find plane_detection_ros)/config/control/position_control_sim.params.yaml" />

    <rosparam
        command="load"
        file="$(find plane_detection_ros)/config/control/joy_sim.config.yaml" />

    <!-- Start plane detection ROS -->
    <node
        pkg="plane_detection_ros"
        type="carrot_control_node"
        name="carrot_control_node"
        output="screen">
        
        <!-- Do all the subscriber remaps -->
        <remap from="joy"               to="joy"           />
        <remap from="/sim/odometry"     to="/bebop/odometry_gt" />
        
        <!-- Do all the publisher remaps -->
        <remap from="control_state"     to="bebop/inspection_mode" />
        <remap from="/sim/rpy_thrust"   to="/bebop/att_override"/>
    </node>

</launch>